src/game/scoring.rs

```diff
@@ -3,11 +3,14 @@ use arrayvec::IntoIter;
 use itertools::*;
```

Comment:
Rust devs prefer explicit deps, except maybe the super::* in tests.

Reply:
Updated crate use to be explicit.